src/ui/gameday/plays.rs

```diff
@@ -60,26 +60,31 @@ fn format_plays(game: &GameState, selected_at_bat: Option<u8>) -> Vec<Line> {
             last_inning = Some(current_inning);
         }
 
-        if let Some(line) = build_line(&play.play_result, selected_at_bat) {
+        let team_abbreviations = (game.home_team.abbreviation, game.away_team.abbreviation);
+        if let Some(line) = build_line(&play.play_result, selected_at_bat, team_abbreviations) {
             lines.push(line);
         }
     }
 
     lines
 }
 
-fn build_line(play: &PlayResult, selected_at_bat: Option<u8>) -> Option<Line> {
+fn build_line<'a>(
+    play: &'a PlayResult,
+    selected_at_bat: Option<u8>,
+    team_abbreviations: (&'static str, &'static str),
```

Comment:
I think instead of a tuple it'd be more clear to just pass separate params here, like 

```
    home_team_abbreviation: &'static str,
    away_team_abbreviation: &'static str,
```